### ISSUE TITLE
Lack of error and confirmation messages [Disk performance measurement window]

### ISSUE THREAD

**Issue by dariiing** [xwiki-labs/cryptpad#1791]
- Inside the admin panel (Performance), when trying to set the disk performance measurement window that doesn't contain a valid number, I don't get any type of error. The expected behavior is that I should have a descriptive error message ( eg. 'Please enter a valid number');
- The confirmation message when setting the new value is missing;
- When entering a negative number, the error message is not suggestive ('Error') and should describe the problem (eg. 'Please enter a positive number')
